core-rust-bridge/src/main/java/com/radixdlt/statecomputer/RustStateComputer.java

```diff
@@ -119,4 +130,22 @@ public Result<CommitSummary, InvalidCommitRequestError> commit(CommitRequest com
       commitFunc;
 
   private static native byte[] commit(NodeRustEnvironment nodeRustEnvironment, byte[] payload);
+
+  public String newestProtocolVersion() {
```

Comment:
Is this the newest pending or the current version?

Reply:
newest pending (or enacted if there are no pending)